src/manipulation/wrapMap.js

```diff
@@ -3,12 +3,11 @@ define( function() {
 // We have to close these tags to support XHTML (#13200)
 var wrapMap = {
 
-	// Support: IE9
```

Comment:
I don't know about such restriction, but if you removed that comment, shouldn't you have some grounds for it?